CONTRIBUTING.md

```diff
@@ -110,6 +110,39 @@ pre-commit install
 from the root of the cuDF repository. Now `isort`, `Black`, and `flake8` will be
 run each time you commit changes.
 
+#### C++/CUDA
+
+cuDF uses [`clang-format`](https://clang.llvm.org/docs/ClangFormat.html)
+
+In order to format the C++/CUDA files, navigate to the root (`cudf`) directory and run:
+```
+python3 ./cpp/scripts/run-clang-format.py -inplace
+```
+
+##### Setting up format on save with VS Code
```

Comment:
Since this is an outward facing document, do we want to keep things editor neutral? I understand that lots of  RAPIDS developers are VS Code users and can benefit greatly from this info -- it just feels a bit awkward to have instructions specific to one editor and one workflow.

I don't feel very strongly about this, and I'm 100% not trying to start any wars here :) Just thought I'd make my voice heard as someone with a different dev workflow.

Reply:
+1 to keeping things editor neutral. Bash + pre-commit only if possible. If we want to start a separate doc for nice RAPIDS VSCode plugin usage or whatnot that's fine.

Reply:
I will delete for now and come back and add later if we choose too.